### PR TITLE
Pytest initial

### PR DESCRIPTION
**Pull request by Stranger6667**
Here is initial approach for running test suite with py.test (#463). Based on current test command invocation.
The problem is, that still there are 3 calls instead of 1.
Next goal is to adopt current tests to run on only on corresponding template packs - all tests conditions will go to test modules directly. 
After that it is possible to parametrize some tests to run on multiple template packs. At this moment it will be good to change invocation to make just 1 call of py.test.
So, here is my draft for first few steps of migration to py.test. I'll be glad to hear what do you think about it.
Thanks!

### CODE REVIEW

**Comment by Stranger6667** [comment]
I don't know why (yet), but test suite runs very long... 43 minutes :(

**Comment by carltongibson** [comment]
> 43 minutes :(

Hmmm. Yes. 

Take a look at #456. Compare the 1.7 and 1.8 run times. It must be something to do with the new template system work arounds I introduced. 

My plan is to run the test suite in the profiler and see where the time is going. You're welcome to take that on! :-)

**Comment by Stranger6667** [comment]
Ok, I'll try to investigate :)

**Comment by Stranger6667** [comment]
Done! Seems like now it is reasonable performance :)

**Comment by carltongibson** [comment]
Good work! I love profilers.

**Comment by Stranger6667** [comment]
:thumbsup:
If you want, I can continue in this branch, or create another one, based on this. I saw some PRs related to testing (#449), so there will be conflicting changes. I can adapt it, if they will be in repo before these changes. So, it depends on your plan :)

**Comment by carltongibson** [comment]
Let me review tomorrow. I'll probably say merge this as is.

**Comment by Stranger6667** [comment]
Also, I want to notice, that in this branch and in master each test call for different template pack runs more test, that it is required. Because of class inheritance.
E.g. bootstrap runs 134 tests now and before. But:
```TestFormHelper``` - has 22 tests
```TestBootstrapFormHelper``` - has 4 own tests and 22 from ```crispy_forms.tests.TestFormHelper```
So, these 22 tests runs twice. Same with all inherited test classes and other template packs. In fact bootstrap template pack has 91 related test.
I'll fix it in the next step.

File: Makefile [hunk 1]
```diff
@@ -5,4 +5,5 @@ develop:
 	pip install -q -e .
 
 test: develop
+	pip install pytest pytest-django
```
Comment (carltongibson) [inline Makefile]: These need to go in `requirements.txt`. Then installing them is covered by the `develop` task.

File: setup.py [hunk 2]
```diff
@@ -5,6 +5,8 @@
 
 tests_require = [
```
Comment (carltongibson) [inline setup.py]: See #466

**Comment by carltongibson** [comment]
@Stranger6667 This looks great. I just to pull and run for sanity and then I'll merge. Good work!

**Comment by Stranger6667** [comment]
Clean up is done! Thanks for feedback :)

**Pull request merged**